assets/helpers/contributions.js

```diff
@@ -48,25 +50,31 @@ type Config = {
 const config: Config = {
   ANNUAL: {
     min: 50,
+    spokenMin: 'fifty',
```

Comment:
Just a picky detail, `writtenMin` probably is a better name.

Reply:
The best description I could find for this was "Amount in words" (remember cheques?).  So maybe `minInWords` or `minAmountInWords`?

Reply:
I don't actually remember cheques, I don't think I ever used them, but I take your point 🙂. Updated.